### DISCUSSION TITLE
七月更新 - 定

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1393] [通知]
## 更改记录

- 在分集区域显示滚动条 @Richasy (#1392)

## 🐛 Bug 修复

- 在手势操作结束后恢复视频播放 @Richasy (#1389)
- 在开启循环的情况下不自动切换下一集 @Richasy (#1388)
- 修复媒体传输控件没有自动隐藏的问题 @Richasy (#1387)
- 修复FFmpeg播放的一些问题，并手动集成SMTC @Richasy (#1386)
- 修复FFmpeg视频播放中断的问题 @Richasy (#1383)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2207.7.0'>七月更新 - 定</a>.</em>

**Comment by XHFO** [comment]
大佬NB
